pkg/machinery/config/types/v1alpha1/generate/generate.go

```diff
@@ -428,17 +428,12 @@ func NewInput(clustername, endpoint, kubernetesVersion string, secrets *SecretsB
 		return nil, err
 	}
 
-	var additionalSubjectAltNames []string
+	additionalSubjectAltNames := append([]string(nil), options.AdditionalSubjectAltNames...)
```

Comment:
this is tricky.

before this PR, we were always adding all endpoints (for QEMU clusters that's all CP nodes IPs) as additional cert SANs, this is not needed at all, and this doesn't allow to test probably proper injection of node's IP address as certSAN.

I couldn't really drop this, as it might break other machinery users, so enabling it only for 0.12